src/api/index.js

```diff
@@ -4,11 +4,13 @@ import placements from './placements'
 import sections from './sections'
 import grades from './grades'
 import students from './students'
+import similarStudent from './similar-student'
 
 export default ({ config, db }) => {
 	let api = Router()
 
 	api.use('/placements', placements({ config, db }))
+	api.use('/similar-student', similarStudent({ config, db }))
```

Comment:
Just a thought - `api.use('students/similar', similarStudent({ config, db} ))` might be a little nicer... e.g. having the similar functionality within the `students` endpoint. IF it works